test/parallel/test-process-versions.js

```diff
@@ -61,7 +61,7 @@ assert.match(process.versions.brotli, commonTemplate);
 assert.match(process.versions.llhttp, commonTemplate);
 assert.match(process.versions.node, commonTemplate);
 assert.match(process.versions.uv, commonTemplate);
-assert.match(process.versions.zlib, commonTemplate);
+assert.match(process.versions.zlib, /^\d+\.\d+\.\d+(?:\.\d+)?(?:-.*)?$/);
```

Comment:
```suggestion
assert.match(process.versions.zlib, /^\d+(?:\.\d+){2,3}(?:-.*)?$/);
```

Reply:
For consistency I would prefer to keep it as is, see line 72. I'm ok with a a new PR to shorten both regex.

Reply:
For me, the reason the first regex wasn't shortened that way is that it would complicate it more than anything else. Unlike this one, which is already a little longer, and in my opinion my suggestion is just as consistent as this regex. But I understand and I don't mind a new PR.

Reply:
I'm talking about this one https://github.com/nodejs/node/blob/d81151e100a77a3c80e2a065c6a37f16c385cece/test/parallel/test-process-versions.js#L72 but I have no strong opinion. I will apply your suggestion.

Reply:
Oh, you're completly right, sorry for my misunderstanding.